src/.gitignore

```diff
@@ -1 +0,0 @@
-*.o
```

Comment:
This file hides all the `.o` files from a build when running `git` commands. Without it, won't all the `.o` files clog up `git status` and `git commit` commands? We do not want to accidentally commit `.o` files to this repo. Why was the file removed?

Reply:
Oops. Looks like there's a top-level `.gitignore` that hides `.o`, and this one is redundant. Right?